src/test/java/teammates/test/cases/ui/browsertests/BaseUiTestCase.java

```diff
@@ -20,6 +20,20 @@
 
 public class BaseUiTestCase extends BaseTestCase {
 
+    /** indicates if the test-run is to use GodMode */
+    protected static Boolean enableGodMode = false;
+
+    /**
+     * Checks if the current test-run should use godmode, 
+     * if yes, enables GodMode
+     */
+    @BeforeSuite
+    public static void checkAndEnableGodMode() {
+        if (enableGodMode) {
+            System.setProperty("godmode", "true");
+        }
+    }
+
```

Comment:
Isn't this the same as `-Dgodmode=true` as VM argument? I don't think it's necessary to be translated to Java code.